frontend/src/scenes/onboarding/sdks/sdk-install-instructions/flutter.tsx

```diff
@@ -17,7 +19,21 @@ function FlutterInstallSnippet(): JSX.Element {
     return <CodeSnippet language={Language.YAML}>posthog_flutter: ^5.0.0</CodeSnippet>
 }
 
-function InstallFlutterSessionReplay(props: FlutterInstallProps): JSX.Element {
+function FlutterDartSetup(props: FlutterSetupProps & FlutterInstallProps): JSX.Element {
```

Comment:
can we merge https://github.com/PostHog/posthog/pull/35487 first otherwise its just gonna be harder to get these changes in

Reply:
or pull my changes to your PR? if needed